### DISCUSSION TITLE
Does SqlSyncChangeTrackingProvider need to create some tables?

### DISCUSSION THREAD

**Discussion by karolgt1** [Mimetis/Dotmim.Sync#838] [Q&A] [answered]
Hi, i'm trying to start working with Dtomim.Sync by using SqlSyncChangeTrackingProviders. Change tracking was enabled on my databases. When i was trying to sync them i got an error: 'CREATE TABLE permission denied in database'. So my question is if i'm using SqlSyncChangeTrackingProviders, will it need to create some tables or other objects on my db? 

Edit. Before using SqlSyncChangeTrackingProviders i was also trying to use SqlSyncProviders, maybe it is important information

**Comment by Mimetis** [comment, marked as answer]
DMS needs to create 2 tables on each side (client / server) called by default `scope_info` and `scope_info_client`
DMS needs to create stored procedures and table value parameters type as well

**Reply by karolgt1** [reply]
Ok, after debugging found that it is creating 'scope_info' table. Is it obligatory? Can we use it without any changes on our databes?

**Reply by Mimetis** [reply]
nop
